paddle/fluid/framework/details/build_strategy.h

```diff
@@ -73,6 +73,8 @@ struct BuildStrategy {
 
   bool fuse_broadcast_op_{false};
 
+  int num_trainers_{1};
+  int trainer_id_{0};
```

Comment:
We don't need this option, let the sort and add deps all the time, this should not impact the performance

Reply:
Sort and add deps all the time will impact performance on sinlge machine.